storm-server/src/main/java/org/apache/storm/daemon/nimbus/TopologyResources.java

```diff
@@ -37,7 +37,7 @@
     private double assignedNonSharedMemOffHeap;
     private double assignedCpu;
     private TopologyResources(TopologyDetails td, Collection<WorkerResources> workers,
-                              Map<String, Double> sharedOffHeap) {
+                              Map<String, Double> sharedOffHeapNode) {
```

Comment:
Nit: Would be good to rename so it's obvious what is being mapped here. Is it node id -> offheap?